MISSING_SEQUENCE_DETECTION.md

```diff
@@ -0,0 +1,117 @@
+# Missing interchange sequence detection
+
+In rare occasion of an in-code exception there is a chance, 
+that a sequence number will be generated and registered in NHAIS database as used
+but due to the error no message will be sent with that number.
+
+Such situation will produce gaps in sequence numbers sent to HA
+
+This document describes how to produce a report on missing sequence numbers.
+
+## Database
+
+NHAIS is using MongoDb as persistence store. 
+This database records metadata of all sent messages including key sequence numbers:
+
+- Interchange Sequence Number (SIS)
+- Message Sequence Number (SMS)
+- Transaction Number (TN)
+
+Database schema information (only relevant fields listed):
+
+    collection: 
+        outboundState
+    fields:
+        sender
+        recipient
+        transactionTimestamp
+        interchangeSequence
+        messageSequence
+        transactionId
+         
+## Report
```

Comment:
I'm realising that we'll also need how to document:

- gaps with inbound sequence numbers
- outbound messages without receps

That can be a further PR though

Reply:
I'll add those requirements to JIRA

Reply:
HA! I see you've already added that :-)